doc/rofi.1.markdown

```diff
@@ -543,6 +543,12 @@ See *PATTERN*.
 
 Default: *{cmd}*
 
+Example to run applications in a dedicated cgroup with systemd:
+
+```
+systemd-run --user --scope --unit='{cmd}-$(date +%s)' {cmd}
```

Comment:
The hierarchy looks like this with the given command:

├─user.slice
  └─user-1000.slice
    ├─user@1000.service
      ├─session.slice
        ├─app.slice
          └─firefox-1668960502.scope